chat/models.py

```diff
@@ -51,6 +51,9 @@ class ChatRoom(models.Model):
     text_channel_id = models.CharField(max_length=255, null=True, blank=True)
     audio_channel_id = models.CharField(max_length=255, null=True, blank=True)
 
+    text_invite_url = models.URLField(blank=True)
+    audio_invite_url = models.URLField(blank=True)
```

Comment:
Same comment as earlier: should we name these as X_channel_url to match the fields above? Also might be good to group them by text/audio.